packages/rich-text/src/test-utils/jsx.ts

```diff
@@ -0,0 +1,33 @@
+import { createHyperscript } from 'slate-hyperscript';
+import { createText } from '@udecode/plate-test-utils';
+import { BLOCKS, INLINES } from '@contentful/rich-text-types';
+
+/**
+ * Mapping for JSX => Slate Node types
+ *
+ * Add items as needed.
+ */
+export const jsx = createHyperscript({
+  elements: {
```

Comment:
Are we missing HR?

Reply:
What about marks? Not necessary?

Reply:
Yeah, some items are missing and I'm still not sure how to add marks. But I'm currently working on that to add more tests

Reply:
I added support for marks with the `htext` tag. You can do `<htext bold> my text </htext>`. I had to add `h` prefix to all tags to prevent colliding with HTML tags for which TypeScripts complains about non-standard attributes

Reply:
I will introduce the other items later when integrating the `slate-test-utils`.